sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigtable/BigtableIO.java

```diff
@@ -1061,7 +1042,7 @@ public RowFilter getRowFilter() {
     }
 
     public String getTableId() {
```

Comment:
Even though this method is public, the class it is in isn't.

We should update this method to return the ValueProvider<String> and have callers deal with it instead of returning null or drop this method and migrate all current callers to `config.getTableId()`

Reply:
Done